src/portal/src/i18n/lang/ja-jp-lang.json

```diff
@@ -0,0 +1,1596 @@
+{
+    "APP_TITLE": {
+        "VMW_HARBOR": "Harbor",
+        "HARBOR": "Harbor",
+        "VIC": "vSphere Integrated Containers",
+        "MGMT": "Management",
+        "REG": "Registry",
+        "HARBOR_SWAGGER": "Harbor Swagger",
+        "THEME_DARK_TEXT": "ダークモード",
+        "THEME_LIGHT_TEXT": "ライトモード"
+    },
+    "SIGN_IN": {
+        "REMEMBER": "ログイン状態を記憶",
+        "INVALID_MSG": "ユーザー名またはパスワードが間違っています。",
+        "FORGOT_PWD": "パスワードを忘れた",
+        "HEADER_LINK": "サインイン",
+        "CORE_SERVICE_NOT_AVAILABLE": "コアサービスが利用不可になっています。"
```

Comment:
Suggested translation: コア サービスを使用できません。
Comment: It would be also better if a space could be added for Katakana compound words to match English words to align with VMware products.